src/scf/chi0models.jl

```diff
@@ -20,23 +17,19 @@ For details see Herbst, Levitt 2020 arXiv:2009.01665
 struct LdosModel <: χ0Model end
 function (::LdosModel)(basis; eigenvalues, ψ, εF, kwargs...)
     n_spin = basis.model.n_spin_components
-    dVol   = basis.model.unit_cell_volume / prod(basis.fft_size)
 
     # Catch cases that will yield no contribution
     iszero(basis.model.temperature) && return nothing
-    ldos = [compute_ldos(εF, basis, eigenvalues, ψ, spins=[σ]) for σ in 1:n_spin]
-    if maximum(maximum(abs, ldos[σ]) for σ in 1:n_spin) < eps(eltype(basis))
+    ldos = compute_ldos(εF, basis, eigenvalues, ψ)
+    if maximum(abs, ldos) < eps(eltype(basis))
         return nothing
     end
 
-    dos = sum(sum, ldos) * dVol  # Integrate LDOS to form total DOS
+    dos = sum(sum, ldos) * basis.dvol
```

Comment:
one tiny inconsistency I see now is that dos / ldos sometimes mean the total dos / ldos and sometimes the spin-adapted. Perhaps we should use tdos / dos and tldos / ldos to distinguish that?